processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -33,10 +34,6 @@
   @NotNull
   private boolean searchCurrentClassloader = true;
 
-  @JsonProperty
-  @NotNull
-  private List<String> coordinates = ImmutableList.of();
```

Comment:
I believe it's sufficient to mention this deprecation in the release note? Even if people set this property, it will be ignored and not cause any harm.